doc/diagrams/sequence/apimethodcallprocedure.puml

```diff
@@ -0,0 +1,79 @@
+@startuml
```

Comment:
This file seems to be an old leftover, right?

Reply:
removed